test_ai.py

```diff
@@ -0,0 +1,29 @@
+from board import BoardState, Board
+from ai import Ai
+
+def test_Ai_only_selects_an_empty_spot_on_board():
+   game_board = Board()
+   game_board.board_state = [
+                                "O", "O", "X",
+                                "X", "O", "O",
+                                "", "X", "X",
+                             ]
+   assert Ai.make_move(game_board.board_state) == 7 
+                                                                   
+def test_Ai_makes_no_move_if_board_is_full():
```

Comment:
I don't understand this test. wouldn't the game be over if the game board is full?